tools/torchci/check_alerts.py

```diff
@@ -220,6 +220,30 @@ def fetch_alerts_filter(repo: str, branch: str, labels: List[str]) -> List[Any]:
     ]
 
 
+def close_if_too_many_comments(issue: Dict[str, Any], dry_run: bool) -> bool:
+    """Close the issue if it has too many comments. Return True if there are too many comments."""
+    if issue["comments"]["totalCount"] > SOFT_COMMENT_THRESHOLD:
```

Comment:
Nit: Does lint suggest combining the 2 if statements into `if issue["comments"]["totalCount"] > SOFT_COMMENT_THRESHOLD and not issue["closed"]`

Reply:
No, theres a return false/true at the end that depends on the outer if statement